wdl-lint/src/rules/call_input_spacing.rs

```diff
@@ -0,0 +1,171 @@
+//! A lint rule for spacing of call inputs.
+
+use wdl_ast::v1::CallStatement;
+use wdl_ast::AstNode;
+use wdl_ast::Diagnostic;
+use wdl_ast::Diagnostics;
+use wdl_ast::Span;
+use wdl_ast::SyntaxKind;
+use wdl_ast::ToSpan;
+use wdl_ast::VisitReason;
+use wdl_ast::Visitor;
+
+use crate::Rule;
+use crate::Tag;
+use crate::TagSet;
+
+/// The identifier for the input not sorted rule.
+const ID: &str = "CallInputSpacing";
+
+/// Creates a input spacing diagnostic.
+fn call_input_spacing(span: Span) -> Diagnostic {
+    Diagnostic::warning("input not properly spaced")
+        .with_rule(ID)
+        .with_highlight(span)
+        .with_fix(
+            "input keyword must be separated from the opening brace (\"{\") by a single space"
+                .to_string(),
+        )
+}
+
+/// Creates an input keyword preceding newline diagnostic.
+fn call_input_keyword_preceding_newline(span: Span) -> Diagnostic {
+    Diagnostic::warning("input keyword may not be preceded by a newline")
+        .with_rule(ID)
+        .with_highlight(span)
+        .with_fix(
+            "input keyword must be separated from the opening brace (\"{\") by a single space \
+             with no newlines",
+        )
+}
+
+/// Creates an input call spacing diagnostic.
+fn call_input_missing_newline(span: Span) -> Diagnostic {
+    Diagnostic::warning("call inputs must be separated by newline")
+        .with_rule(ID)
+        .with_highlight(span)
+        .with_fix("add newline before the input")
+}
+
+/// Creates call input assignment diagnostic.
+fn call_input_assignment(span: Span) -> Diagnostic {
+    Diagnostic::warning("call inputs assignments must be surrounded with whitespace")
+        .with_rule(ID)
+        .with_highlight(span)
+        .with_fix("surround '=' with whitespace on each side")
+}
+
+/// Detects unsorted input declarations.
+#[derive(Debug, Clone, Copy)]
+pub struct CallInputSpacingRule;
+
+impl Rule for CallInputSpacingRule {
+    fn id(&self) -> &'static str {
+        ID
+    }
+
+    fn description(&self) -> &'static str {
+        "Ensures that call inputs are spaced appropriately."
+    }
+
+    fn explanation(&self) -> &'static str {
+        "When making calls from a workflow, it is more readable and easier to edit if the supplied \
+         inputs are each on their own line. This does inflate the line count of a WDL document, \
```

Comment:
```suggestion
        "When making calls from a workflow, it is more readable and easier to edit if the supplied \
         inputs are each on their own line. <More details here.> This does inflate the line count of a WDL document, \
```
Needs the specifics about how whitespace should work in these statements.